src/Symfony/Component/Form/Tests/AbstractDivLayoutTest.php

```diff
@@ -298,7 +298,9 @@ public function testCollection()
         );
     }
 
-    // https://github.com/symfony/symfony/issues/5038
+    /**
+     * Test for issue https://github.com/symfony/symfony/issues/5038.
+     */
```

Comment:
Can you remove all comments with issues reference as we don't need them? Git already has all needed information.